dart/common/Deprecated.hpp

```diff
@@ -42,7 +43,7 @@
 //==============================================================================
 
 #if defined(__GNUC__) || defined(__clang__)
-  #define DART_DEPRECATED(version) __attribute__ ((deprecated))
+  #define DART_DEPRECATED(version) DETAIL_DART_DEPRECATED
   #define DART_FORCEINLINE __attribute__((always_inline))
 #elif defined(_MSC_VER)
   #define DART_DEPRECATED(version) __declspec(deprecated)
```

Comment:
This should also be redirected to `DETAIL_DART_DEPRECATED` to be consistent with the other definition.